libgalois/include/katana/PropertyGraph.h

```diff
@@ -66,6 +66,9 @@ struct KATANA_EXPORT GraphTopology {
     return MakeStandardRange<edge_iterator>(begin_edge, end_edge);
   }
   Node edge_dest(Edge eid) const {
+    if (eid >= Edge(out_dests->length())) {
+      KATANA_LOG_WARN("eid = {}, out_dests length = {}", eid, out_dests->length());
+    }
     KATANA_LOG_ASSERT(eid < static_cast<Edge>(out_dests->length()));
```

Comment:
Just noting it here as well. Please change this to debug assert. and drop the log warn before you merge.

Reply:
done.